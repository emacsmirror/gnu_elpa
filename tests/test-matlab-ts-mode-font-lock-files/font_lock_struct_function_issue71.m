% -*- matlab-ts -*-

% See; https://github.com/acristoffers/tree-sitter-matlab/issues/71

y = 7;
fh = @(x)x.^2+y;
s = functions(fh)
s.function

