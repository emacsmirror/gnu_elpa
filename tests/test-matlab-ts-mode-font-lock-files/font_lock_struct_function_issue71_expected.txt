c ccc ccccccccc ccc

c cccc cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc

v o nD
vv o obdbdoonodD
v o BBBBBBBBBbddb
dDdddddddd

